tests/test_config.py

```diff
@@ -732,33 +732,48 @@ def test_apply_config(self):
 
         # Succeed when calling the apply_config function with a valid
         # Upload API object and config
-        upload_api = UploadAPI(app_id="31337")
+        with patch("veracode.api.get_app_id", return_value="1337"):
+            upload_api = UploadAPI(app_name="TestApp")
+
         applied_upload_api = config.apply_config(api=upload_api, config=configuration)
         self.assertEqual(applied_upload_api, upload_api)
 
         # Succeed when calling the apply_config function with a valid
         # Results API object and config
-        results_api = ResultsAPI(app_id="31337")
+        with patch("veracode.api.get_app_id", return_value="1337"):
+            results_api = ResultsAPI(app_name="TestApp")
+
         applied_results_api = config.apply_config(api=results_api, config=configuration)
         self.assertEqual(applied_results_api, results_api)
 
         # Succeed when calling the apply_config function with a valid
         # Sandbox API object and config
-        sandbox_api = SandboxAPI(
-            app_id="31337", sandbox_name="easy_sast/fb/jonzeolla/testing"
-        )
-        applied_sandbox_api = config.apply_config(api=sandbox_api, config=configuration)
-        self.assertEqual(applied_sandbox_api, sandbox_api)
+        with patch("veracode.api.get_app_id", return_value="1337"):
+            sandbox_api = SandboxAPI(
+                app_name="TestApp", sandbox_name="easy_sast/fb/jonzeolla/testing"
+            )
+            applied_sandbox_api = config.apply_config(
+                api=sandbox_api, config=configuration
+            )
+            self.assertEqual(applied_sandbox_api, sandbox_api)
 
         # Ensure calling the apply_config function with different `app_id`s
         # does not result in an object with the same version string (which is
         # unique per-API)
-        upload_api = UploadAPI(app_id="31337")
-        applied_upload_api = config.apply_config(api=upload_api, config=configuration)
-        # Note: This must be different than the above app_id
-        results_api = ResultsAPI(app_id="1337")
-        applied_results_api = config.apply_config(api=results_api, config=configuration)
-        self.assertNotEqual(applied_results_api.version, applied_upload_api.version)
+        with patch("veracode.api.get_app_id", return_value="1337"):
```

Comment:
See the comment below `# Note: This must be different than the above app_id`
```suggestion
        with patch("veracode.api.get_app_id", return_value="31337"):
```